step-generation/src/fixtures/robotStateFixtures.ts

```diff
@@ -149,7 +149,7 @@ export function makeContext(): InvariantContext {
     p10SingleId: {
       name: 'p10_single',
       id: 'p10SingleId',
-      pythonName: 'mockPythonName',
+      pythonName: 'pipette_p10',
```

Comment:
nit, can this pythonName start with `mock_`, since its a fixture?
```suggestion
      pythonName: 'mock_pipette_p10',
```

Reply:
OK!